webroot/js/snodas_map.js

```diff
@@ -1169,3 +1207,24 @@ if (!L.Browser.touch) {
 } else {
     L.DomEvent.disableClickPropagation(container);
 }
+
+function selectMapLyr()
+{
+    var x = document.getElementById("awdb-select");
+    if (x.value != null)
+    {
+        var lyr = pourpoints.getLayerByID(x.value);
+        lyr.fireEvent('click');
+    }
+
+}
+
+function setOptions(opts) {
+	var select = document.getElementById("awdb-select");
+	//set values for drop down
+	var html = '';
+	opts.forEach(o => {
+		html += `<option value=${o.pourpoint_id}>${o.awdb_id.concat(":".concat(o.name))}</option>`;
+	});
+	select.innerHTML = html;
+}
```

Comment:
Looks like you got some tabs in here instead of spaces? While js doesn't care, it's probably good to try to be consistent. I think there may be some more earlier in the file too.